### PR TITLE
BNI Intelligence Officer

### PR DESCRIPTION
**Pull request by thevandie**
## About The Pull Request

Adds the entirety of the BNI Intelligence Agent's outfit. He has a rank pin, a chiappo, and most important of all, a full BNI uniform. Very Fashionable.

## Why It's Good For The Game

New outfits for outfit-less senior titles is neat.

## Changelog
:cl:
add: BNI Intelligence officer senior title outfit
/:cl:

bniagent2

### CODE REVIEW

File: ModularBungalow/altjobtitles/outfits.dm [hunk 1]
```diff
@@ -93,6 +93,22 @@
...
+	name = "Detective (Intelligence Officer)"
+	head = /obj/item/clothing/head/beret/sec/bni
+	uniform = /obj/item/clothing/under/bni
+	neck = null
+	suit = /obj/item/clothing/suit/det_suit/noir/intel
+	accessory = /obj/item/clothing/accessory/medal/rank/nt/bnisnr
+	suit_store = /obj/item/gun/ballistic/revolver/rhino
+	shoes = /obj/item/clothing/shoes/jackboots
+	gloves = /obj/item/clothing/gloves/color/black
+	r_hand = /obj/item/encryptionkey/intel
```
Comment (Kitsunemitsu) [inline ModularBungalow/altjobtitles/outfits.dm:107]: Gameplay changes, Please revise.

File: ModularBungalow/altjobtitles/outfits.dm [hunk 2]
```diff
@@ -93,6 +93,22 @@
...
+	backpack = /obj/item/storage/backpack/security
```
Comment (Kitsunemitsu) [inline ModularBungalow/altjobtitles/outfits.dm:108]: 108-110 are unnecessary.

File: ModularBungalow/altjobtitles/outfits.dm [hunk 3]
```diff
@@ -93,6 +93,22 @@
...
+/datum/outfit/job/detective/intelligenceofficer
+	name = "Detective (Intelligence Officer)"
+	head = /obj/item/clothing/head/beret/sec/bni
+	uniform = /obj/item/clothing/under/bni
+	neck = null
+	suit = /obj/item/clothing/suit/det_suit/noir/intel
+	accessory = /obj/item/clothing/accessory/medal/rank/nt/bnisnr
+	suit_store = /obj/item/gun/ballistic/revolver/rhino
+	shoes = /obj/item/clothing/shoes/jackboots
+	gloves = /obj/item/clothing/gloves/color/black
```
Comment (Kitsunemitsu) [inline ModularBungalow/altjobtitles/outfits.dm:106]: Not needed.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Changes requested, please revise before merge

File: ModularBungalow/altjobtitles/outfits.dm (hunk 2 above)
Comment (thevandie) [reply ModularBungalow/altjobtitles/outfits.dm:107]: got rid of the duffels and stuff

**Review by Kitsunemitsu — APPROVED** [review]

**Pull request merged**